server/mongodb/actions/meeting.ts

```diff
@@ -0,0 +1,27 @@
+import { connectToDB } from "../index";
+import MeetingSchema from "../Meeting";
+import { ObjectId } from "mongodb";
+
+export async function addMeeting(meeting: Document) {
+  await connectToDB();
+
+  return MeetingSchema.create(meeting);
+}
+
+export async function getMeetings() {
+  await connectToDB();
+
+  return MeetingSchema.find().sort({ startDateTime: -1 });
+}
+
+export async function getMeetingById(id: ObjectId) {
+  await connectToDB();
+
+  return MeetingSchema.findOne({ _id: id });
+}
+
+export async function getMeetingByApplication(id: ObjectId) {
+  await connectToDB();
+
+  return MeetingSchema.findOne({ _application: id });
```

Comment:
Replace this with `{ application: new ObjectId(id) }`, `application` in the model does not include an underscore, and it's likely that `id` will actually be a string so we want to cast it to an `objectId` before the query.